Analytics-CSharp/Segment/Analytics/Types.cs

```diff
@@ -27,16 +27,7 @@ internal void ApplyRawEventData(RawEvent rawEvent)
             this.timestamp = rawEvent.timestamp;
             this.context = rawEvent.context;
             this.integrations = rawEvent.integrations;
-        }
-
-        internal async Task ApplyRawEventData(Store store)
-        {
-            var userInfo = await store.CurrentState<UserInfo>();
-            if (userInfo.isNull) return;
-
-            this.anonymousId = userInfo.anonymousId;
-            this.userId = userInfo.userId;
-            this.integrations = new JsonObject();  
+            this.integrations = new JsonObject();
         }
 
         internal void ApplyBaseData()
```

Comment:
we should merge `ApplyBaseData()` and `ApplyRawEventData(store)` to just `ApplyRawEventData()`. `ApplyBaseData` was separate from `ApplyRawEventData` because of the async operation. now we don't have async operation anymore, we can merge them back

Reply:
Since the IDs now come from the plugin and there's no analytics here, ApplyRawEventData is mostly gutted so I like moving the integrations set to ApplyBaseData and keeping that name.